docs/rules/prefer-async-await.md

```diff
@@ -0,0 +1,31 @@
+# Prefer using async/await instead of returning a Promise
+
+Translations: [Français](https://github.com/avajs/ava-docs/blob/master/fr_FR/related/eslint-plugin-ava/docs/rules/prefer-async-await.md)
+
+AVA comes with built-in support for async functions (async/await). This allows you to write shorter and clearer tests.
```

Comment:
Can you linkify `async functions` to http://www.2ality.com/2016/02/async-functions.html